### PR TITLE
Add a new feature called 'altpage' to configuration section

### PR DESCRIPTION
**Pull request by sindhuragrandhi**
Signed-off-by: Sindhura Grandhi <sindhura.grandhi@intel.com>

### CODE REVIEW

File: source/specs/config.rst [hunk 1]
```diff
@@ -871,6 +871,39 @@ Format::
...
+               Enable or Disable Hyper Threading; 0- Disable; <b>1- Enable</b>
+
+      length : 0x01
+
+      value  : 0x1
+
+
+HyperThreading is a configuration option which is defined under MEMORY_CFG_DATA. The same can appear in a different page say "TCCSETTINGS"
+with the help of keyword 'altpage' as shown above. The same value will be reflected at both places and when changed in one page, the changed value is reflected in the alternate page as well.
+
```
Comment (mauricema) [inline source/specs/config.rst:906]: Maybe add some info on why it could be useful ?  EX:  Group options from different view perspective, etc.

File: source/specs/config.rst [hunk 2]
```diff
@@ -871,6 +871,39 @@ Format::
...
+Altpage
+^^^^^^^
+
+Alternate page allows a specific configuration item to appear at multiple pages as desired. Lets describe this using an example.
+
+::
+
+  - MEMORY_CFG_DATA :
+
+    - !expand { CFGHDR_TMPL : [ MEMORY_CFG_DATA, 0x200, 0, 0 ] }
```
Comment (mauricema) [inline source/specs/config.rst:883]: Let us remove this line since it is not related.
Removing it will make it easier.

File: source/specs/config.rst (hunk 1 above)
Comment (sindhuragrandhi) [reply source/specs/config.rst:904]: I added some info. Please review.

**Review by mauricema — APPROVED** [review]
Looks good to me.
